.samples/websocket-chat-app.yaml

```
@@ -1,5 +1,5 @@
-displayName: WebSocket Chat Application
-description: Provides simultaneous two-way communication over a single TCP connection.
+displayName: WebSocket Chat App in NodeJS
+description: A WebSocket chat application with a Node.js server and a React client.
```

What if we change this to,
```suggestion
description: WebSocket chat application with a Node.js server and a React client.
```